### PR TITLE
Test add remove part 2

### PR DESCRIPTION
**Pull request by tufoinnkuo10**
In this part of the project :
- we followed the AAA format
- imported functions to be tested
- used barbel to enable the import and export
- wrote their description for the add and remove test functions
- used jest coverage to test it
- added the mock and dom
- used the jsdom environment to check the dom
- tested edit task, update complete task and clear all task
- update read me and add a screenshot of all tests being succesful
- Add procedure to get it running

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ To do list
 > This project is designed to display a list of activities to be fufiled in a day. it is made possible with the use of webpack
 > ![screenshot](./image/todo.png)
 
+A picture of the test ran on the todo 
+> ![screenshot](./image/testall.png)
+
```
Comment (mikenath223) [inline README.md:12]: - [x] Kindly include the Getting started section in the readme and add instructions to both get the project started and also run tests.

**Comment by mikenath223** [comment]
- [ ] OPTIONAL - Going forward we can make better commit messages, capitalize the first word in the commit message. Also, commit messages shouldn't be as short as 2 words it's not really descriptive that way.

File: README.md (hunk 1 above)
Comment (tufoinnkuo10) [reply README.md:12]: Thank you.
 I just updated the readme

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**